app/models/service_template_ansible_playbook.rb

```diff
@@ -50,19 +50,46 @@ def self.prepare_job_template_and_dialog(action, service_name, description, conf
   end
   private_class_method :prepare_job_template_and_dialog
 
-  def self.create_job_template(name, description, info)
-    playbook = ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook.find(info[:playbook_id])
-    # tower = playbook.manager
+  def self.create_job_templates(service_name, description, config_info, auth_user)
+    {}.tap do |hash|
+      [:provision, :retirement, :reconfigure].each do |action|
+        next unless config_info[action]
+        hash[action.to_sym] = { :configuration_template => create_job_template("#{service_name}_#{action}", description, config_info[action], auth_user) }
```

Comment:
Using the ID or UUID of the service_template make sense to me since they are already part of the model.  (Remember that with a region id the ID value will be almost as long as the UUID.)  But if the name length is not a problem in Tower then I do not think we should be too concerned with it here.

If this is the only thing blocking this PR can we open an issue (and/or pivotal story) to address it in a followup PR.  From a discussion with @syncrou it sounds like creating the service_template first will require changes to another existing PR as well.